### DISCUSSION TITLE
Clarify how TTL works (i.e. only when the user ever revisits my website)

### DISCUSSION THREAD

**Discussion by ThomasLandauer** [digitalfortress-tech/localstorage-slim#31] [Ideas]
The term "TTL" (time-to-live) creates the impression that items will actually get deleted after that time. But this is not always the case: If somebody visits my websites, and I save something into the user's LocalStorage, and the user *never* returns to my website, the stuff stays in the user's LocalStorage **forever**.

I think you should clarify this in README.

**Comment by niketpathak** [comment]
Thank you for your comment @ThomasLandauer.  Localstorage is not built for this (I haven't documented it because its already there in the official documentation of LocalStorage). If someone else was able to read or delete your data using this library, that would be bad since its a breach of security/privacy and the localstorage API doesn't allow that. 
To accomplish what you want, you can use something else that supports expiry time by design. for example:  `cookies` .

Hope that helps :)
